Can't deploy environment EC2 into existing VPC

The documentation suggests that the following should allow you to use the Environment stack to deploy an EC2 only, but it fails with the error "Caused by: java.lang.RuntimeException: Error: App at '' should be created in the scope of a Stack, but no Stack found":

    cdk deploy -c instanceId=myenv -c vpcId=myvpc "*-BuildEC2"

--- a/java/cdk-environment/src/main/java/sleeper/environment/cdk/buildec2/BuildEC2Stack.java
+++ b/java/cdk-environment/src/main/java/sleeper/environment/cdk/buildec2/BuildEC2Stack.java
@@ -52,7 +52,7 @@ public class BuildEC2Stack extends Stack {
         AppContext context = AppContext.of(this);
         BuildEC2Parameters params = BuildEC2Parameters.from(context);
         vpc = context.get(VPC_ID)
-                .map(vpcId -> Vpc.fromLookup(scope, "Vpc", VpcLookupOptions.builder().vpcId(vpcId).build()))
+                .map(vpcId -> Vpc.fromLookup(this, "Vpc", VpcLookupOptions.builder().vpcId(vpcId).build()))
                 .orElse(inheritVpc);
         BuildEC2Image image = params.image();
 

